src/main/java/org/elastos/hive/AuthHelper.java

```diff
@@ -3,14 +3,13 @@
 import java.util.concurrent.CompletableFuture;
 
 public interface AuthHelper {
-	public AuthToken getToken();
+	AuthToken getToken();
 
-	public CompletableFuture<Status> 	loginAsync(Authenticator authenticator);
-	public CompletableFuture<Status> 	loginAsync(Authenticator authenticator, Callback<Status> callback);
+	CompletableFuture<AuthToken> loginAsync(Authenticator authenticator);
```

Comment:
Need to sync the last commit. 
CompletableFuture<AuthToken> loginAsync(Authenticator authenticator); ---> 
CompletableFuture<Status> loginAsync(Authenticator authenticator);